demos/python_demos/llm_text_generation/README.md

```diff
@@ -50,46 +106,78 @@ Depending on the use case, `./servable_unary` and `./servable_stream` showcase d
 ```
 
-## Requesting the LLM
+You can also deploy the compressed model by just changing the model path mounted to the container. For example:
+
+```bash
+docker run -d --rm -p 9000:9000 -v ${PWD}/servable_unary:/workspace -v ${PWD}/${SELECTED_MODEL}_INT8_compressed_weights:/model \
+-e SELECTED_MODEL=${SELECTED_MODEL} openvino/model_server:py  --config_path /workspace/config.json --port 9000
+```
+> **Note** Check the docker container logs to confirm the model is loaded before running the client. Depending on the model and hardware it might take up to several minutes.
+
+### Running the client with LLM model and unary gRPC call
+
+Install python client dependencies. This is a common step also for the streaming client.
+```bash
+pip install -r client_requirements.txt
+```
 
 Run time unary client `client_unary.py`:
 ```bash
-python3 client_unary.py --url localhost:9000 --prompt "Describe the state of the healthcare industry in the United States in max 2 sentences"
+python3 client_unary.py --url localhost:9000 --prompt "How many helicopters can a human eat in one sitting?"
 ```
 
 Example output:
 ```bash
 Question:
-Describe the state of the healthcare industry in the United States in max 2 sentences
+How many helicopters can a human eat in one sitting?
 
 Completion:
- Many jobs in the health care industry are experiencing long-term shortages due to a lack of workers, while other areas face overwhelming stress and strain.  Due to COVID-19 many more people look for quality medical services closer to home so hospitals have seen record levels of admissions over the last year.
+It is difficult to say how many helicopters human can eat in one sitting without knowing what type of person you are referring to. You may want to ask someone who knows about this topic for an accurate response to this question. However, typically speaking, it would be impossible for a human to consume an entire aerial vehicle, consisting of multiple compartments and rotors, every day if they lived to be 100 years old. However, humans can ingest larger quantities of food, like energy bars or canned goods, which have a smaller volume and can be consumed over a period of time, making it easier for them to consume large amounts of food at once. It is also possible that some people are able to consume helicopter parts due to their exceptional strength, stamina, endurance, or aversion to dehydration, among other reasons.
+
+Total time 11662 ms
 ```
 
-## Requesting the LLM with gRPC streaming
+## Use LLM model with gRPC streaming
 
+### Deploy OpenVINO Model Server with the Python Calculator
 
-Start the Model Server with different directory mounted (`./servable_stream`).
+The model server can be deployed with our streaming example by just mounting different workspace location from `./servable_stream`.
 It contains modified `model.py` script which yields the intermediate results instead of returning it at the end of `execute` method.
```

Comment:
```suggestion
It contains a modified `model.py` script which provides intermediate results instead of returning the full result at the end of the `execute` method.
```